src/emailalert/email_function.py

```diff
@@ -9,11 +10,11 @@ def lambda_handler(event, context):
     SUBJECT = "Failure alert for RDS Restore Pipeline"
     result = {}
     if 'status' in event:
-        result['Error'] = event['taskname'] + 'Error'
-        result['Cause'] = event['status']
```

Comment:
the key given in the constants file is 'ERROR' and 'CAUSE'. 
also do replace it on the `event['taskname'] + 'Error'` as well